pkgs/dart_flutter_team_lints/lib/analysis_options.yaml

```diff
@@ -47,18 +51,22 @@ linter:
     - unnecessary_statements
     - use_is_even_rather_than_modulo
     - use_string_in_part_of_directives
+    - use_super_parameters
 
     # correctness
     - always_declare_return_types
     - avoid_catching_errors
     - avoid_dynamic_calls
     - await_only_futures
+    - cancel_subscriptions
```

Comment:
I don't think this lint is reliable enough - there are false positives.

Reply:
sgtm - will remove